pkg/k8s/client.go

```diff
@@ -84,3 +86,31 @@ func (s *LazyClient) initialize() {
 		panic(s.err())
 	}
 }
+
+func New(clusterID string, options client.Options) (client.Client, error) {
```

Comment:
**thought**: This backplane client usage here is something we're definitely going to implement into CAD. I opened https://issues.redhat.com/browse/OSD-17159 for this and referenced to the osdctl implementation :)